### PR TITLE
database migration for team realationships

### PR DESCRIPTION
**Pull request by botany-lead**
database migration for team realationships

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ new TOS page
 new mention handler tests
 
 add tests for new auth redirect
+
+database migration for team realationships
```
Comment (botany-demo) [inline readme.md:21]: this is well designed. should cover all the possible cases

**Review by botany-demo — APPROVED** [review]
looks great 👍

**Comment by botany-lead** [comment]
testing @botany-dummy-org-1/team_1

**Comment by botany-lead** [comment]
testing again @botany-dummy-org-1/team_1

**Pull request merged**